src/api/app/policies/comment_policy.rb

```diff
@@ -0,0 +1,25 @@
+class CommentPolicy < ApplicationPolicy
+  def initialize(user, record)
```

Comment:
I would say that you don't need that method, as it is in` application_policy.rb.` The only difference will be that if the user is not logged in it will raise the exception there instead of in the comment controller. But in my opinion this is even better as the error will be _must be logged in_ instead of _No permissions to delete comment_